### DISCUSSION TITLE
Sample projects won't run

### DISCUSSION THREAD

**Discussion by MrSharpFox** [microsoft/WindowsAppSDK#3444] [Q&A]
Hi,

I try to run the Notifications project 'CsUnpackagedAppNotifications' from the SDK samples but I don't get it to run. Within VS 2022 I don't get any errors (while building or running) but the app won't start. But I found the following exception in the Windows Event viewer:

Windows App Runtime 
   ERROR 0x80070005: Bootstrapper initialization failed while looking for version 1.2 (MSIX package version >= 2000.747.1945.0) 

But the latest version of the SDK has been installed:

![image](https://user-images.githubusercontent.com/125543041/219299169-b41bcc25-252e-47c0-9bdd-c8b3af92ae19.png)

Any help is welcom,

Thanks!

**Comment by DarranRowe** [comment]
That is looking for the dynamic dependencies package, it is possible for the runtime to be installed but not the dynamic dependencies package. This can happen when you install a packaged application, it downloads the runtime package but not the rest.
Your best bet is to get the redistributable and install that, or if you are feeling adventurous then you can use the MSIX files distributed with the Windows App SDK NuGet package to fill in the blanks. If you do that, prefer your system's native architecture over others.

**Comment by MrSharpFox** [comment]
I have installed the redistributable and the msix files but I still can't get it to work and keep on getting the same exception in the event viewer.

I read and followed https://learn.microsoft.com/nl-nl/windows/apps/windows-app-sdk/deploy-unpackaged-apps but it didn't bring me any further.

When I check for the installed packages it tells me the following:
`powershell -c $(get-appxpackage micro*win*appruntime*).packagefullname
Microsoft.WindowsAppRuntime.1.2_2000.747.1945.0_x86__8wekyb3d8bbwe
Microsoft.WinAppRuntime.DDLM.2000.747.1945.0-x8_2000.747.1945.0_x86__8wekyb3d8bbwe
Microsoft.WinAppRuntime.DDLM.2000.747.1945.0-x6_2000.747.1945.0_x64__8wekyb3d8bbwe
Microsoft.WindowsAppRuntime.1.2_2000.747.1945.0_x64__8wekyb3d8bbwe
MicrosoftCorporationII.WinAppRuntime.Main.1.2_2000.747.1945.0_x64__8wekyb3d8bbwe
MicrosoftCorporationII.WinAppRuntime.Singleton_2000.747.1945.0_x64__8wekyb3d8bbwe`

**Reply by DarranRowe** [reply]
Unfortunately, I'm just not seeing this at all.

![Screenshot 2023-02-17 214948](https://user-images.githubusercontent.com/52577874/219801699-ab835ce6-eec4-4ebd-a303-da85c257f080.png)

The x64 version runs perfectly fine. The x86 version crashes, but it crashes in Microsoft.UI.Xaml.dll, so it obviously managed to bootstrap correctly.
